llvm/lib/Transforms/Utils/ModuleUtils.cpp

```diff
@@ -208,10 +209,34 @@ void llvm::setKCFIType(Module &M, Function &F, StringRef MangledType) {
   std::string Type = MangledType.str();
   if (M.getModuleFlag("cfi-normalize-integers"))
     Type += ".normalized";
+
+  uint32_t OutHash = static_cast<uint32_t>(llvm::xxHash64(Type));
+  auto T = Triple(Twine(M.getTargetTriple()));
```

Comment:
This looks awkward and I regret that I needed to include another header to enable this. Maybe there is a workable API within the existing headers, but I couldn't find one.

Reply:
Like this? https://github.com/llvm/llvm-project/blob/main/llvm/lib/Transforms/Utils/EscapeEnumerator.cpp#L24

Reply:
Thank you for the suggestion! This does look tidier than what I had written. I updated the PR.